Tools/machines/perlmutter-nersc/install_cpu_dependencies.sh

```diff
@@ -109,6 +109,45 @@ CXX=$(which CC) CXXFLAGS="-DLAPACK_FORTRAN_ADD_" cmake -S $HOME/src/lapackpp -B
 cmake --build ${build_dir}/lapackpp-pm-cpu-build --target install --parallel 16
 rm -rf ${build_dir}/lapackpp-pm-cpu-build
 
+# heFFTe
+if [ -d $HOME/src/heffte ]
+then
+  cd $HOME/src/heffte
+  git fetch --prune
+  git checkout v2.4.0
+  git pull
+  cd -
+else
+  git clone -b v2.4.0 https://github.com/icl-utk-edu/heffte.git ${HOME}/src/heffte
+fi
+rm -rf $HOME/src/heffte-pm-cpu-build
+cmake \
+    -S ${HOME}/src/heffte               \
+    -B ${build_dir}/heffte-pm-cpu-build \
+    -DBUILD_SHARED_LIBS=ON              \
+    -DCMAKE_BUILD_TYPE=Release          \
+    -DCMAKE_CXX_STANDARD=17             \
+    -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON  \
+    -DCMAKE_INSTALL_PREFIX=${SW_DIR}/heffte-2.4.0  \
+    -DHeffte_DISABLE_GPU_AWARE_MPI=ON   \
+    -DHeffte_ENABLE_AVX=ON              \
+    -DHeffte_ENABLE_AVX512=ON           \
```

Comment:
Tried this on PM, and it leads to `illegal instruction` errors as soon as heFFTe is linked. So turning AVX512 off:
```suggestion
    -DHeffte_ENABLE_AVX512=OFF          \
```